cmd/build/generate.go

```diff
@@ -45,21 +45,23 @@ func (g Generate) All(ctx context.Context) error {
 }
 
 func (Generate) code() error {
+	const paths = "paths="
+
 	err := shr.New(sh.WithWorkDir("apis")).Run("controller-gen",
 		"crd:crdVersions=v1,generateEmbeddedObjectMeta=true",
-		"paths=./core/...",
-		"output:crd:artifacts:config=../config/crds",
+		paths+dotFilepathJoin("core", "..."),
```

Comment:
Not a file path, only use forward slashes

Reply:
FixedFixed